everviz/config.py

```diff
@@ -9,16 +9,20 @@
 
 
 def webviz_config(api):
+    pages = [
+        {"title": "Everest", "content": [],},
+        controls.page_layout(api),
+        summary_values.page_layout(api),
+        crossplot.page_layout(api),
+        configuration.page_layout(api),
+        objectives.page_layout(api),
+    ]
+
+    pages = list(filter(None, pages))
```

Comment:
LGTM, but perhaps we should add a comment as to why we´re doing this? Perhaps it´s obvious.

Reply:
I agree, a comment will be better!